integreatly/integreatly-1.3.0/integreatly-operator.1.3.0.clusterserviceversion.yaml

```diff
@@ -177,20 +180,23 @@ spec:
                 - create
                 - watch
                 - get
+            #Allow us to create oauth clients (Note could be limited further by using named resources)
             - apiGroups:
                 - oauth.openshift.io
               resources:
                 - oauthclients
               verbs:
                 - create
                 - get
+            #Allow us to get and update configmaps (Note seems overly permissive need to revisit)                
```

Comment:
@philbrookes  wondering why we need it to be this permissive

Reply:
This is part of 3scale as well. Currently we need to get and update a configmap called `system` in whatever namespace 3scale is in. We could limit it by `resourceNames`